### PR TITLE
panlint: Add check for global variable names

### PR DESCRIPTION
**Pull request by jrha**
Check that they have uppercase names longer than five characters
Add unit tests for checks.

### CODE REVIEW

**Comment by ned21** [comment]
I am OK with adding these checks to panlint. I haven't checked the regexes though.

**Comment by jrha** [comment]
This is the first of (hopefully) many PRs that I am pulling out of our rather messy development branch that I mentioned at the workshop, the code is all in use at RAL in one form or another.

File: panc/src/main/scripts/panlint/panlint.py [hunk 1]
```diff
@@ -56,6 +56,8 @@
     "Commas should be followed by exactly one space": re.compile(r'(?P<error>,(?:\S|\s{2,}))'),
     "Whitespace before semicolon": re.compile(r'(?P<error>\s+;)'),
     "Semicolons should be followed exactly one space or end-of-line": re.compile(r';(?P<error>(?:\S|\s{2,}))'),
+    "Global variables should be uppercase": re.compile(r'variable\s+(?P<error>[\w]+)(?<=\S[a-z]\S)'),
+    "Global variables should be five or more characters": re.compile(r'variable (?P<error>\w{1,4})\b'),
```
Comment (wpoely86) [inline panc/src/main/scripts/panlint/panlint.py]: `variable\s+`

**Comment by ned21** [comment]
@wpoely86 you have a stale review, are you happy with this now? If so please go ahead and approve & merge.

**Comment by wpoely86** [comment]
@ned21 I don't have merge rights.

**Comment by ned21** [comment]
@wpoely86 OK, can you given an "approved" and I'll do the merge?

**Pull request merged**